src/v/kafka/latency_probe.h

```diff
@@ -40,6 +40,30 @@ class latency_probe {
              sm::description("Produce Latency"),
              labels,
              [this] { return _produce_latency.seastar_histogram_logform(); })});
+
+        _metrics.add_group(
+          prometheus_sanitize::metrics_name("kafka"),
+          {
+            sm::make_histogram(
+              "request_latency_seconds",
+              sm::description("Internal latency of client request"),
+              {sm::label("request")("produce")},
+              [this] {
+                  return _produce_latency.seastar_histogram_logform(
+                    18, 250, 2.0, 1000000);
+              },
+              {sm::shard_label}),
+            sm::make_histogram(
+              "request_latency_seconds",
+              sm::description("Internal latency of client request"),
+              {sm::label("request")("consume")},
+              [this] {
+                  return _fetch_latency.seastar_histogram_logform(
```

Comment:
It might be worth using this as the named type in `ssx/metrics.h` here as well as for pandaproxy/schema registry.